libs/designsystem/src/lib/components/icon/icon.component.spec.ts

```diff
@@ -47,6 +47,22 @@ describe('IconComponent', () => {
       expect(component.name).toBe(undefined);
       expect(component.icon.name).toBe(component.defaultIcon.name);
     });
+
```

Comment:
Consider adding missing sunshine test; `it('should use default icons from Kirby icon settings'` ...

Reply:
added